### ISSUE TITLE
Internal problem

### ISSUE THREAD

**Issue by kasitif** [spyder-ide/spyder#21506]
### What steps will reproduce the problem?

Here is an error that I have ncountered. During analysis period

### Traceback
```python-traceback
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\radar\lib\site-packages\spyder\plugins\editor\utils\autosave.py", line 135, in do_autosave
    stack.autosave.autosave_all()
  File "C:\ProgramData\Anaconda3\envs\radar\lib\site-packages\spyder\plugins\editor\utils\autosave.py", line 417, in autosave_all
    self.maybe_autosave(index)
  File "C:\ProgramData\Anaconda3\envs\radar\lib\site-packages\spyder\plugins\editor\utils\autosave.py", line 381, in maybe_autosave
    autosave_hash = self.file_hashes[autosave_filename]
KeyError: 'C:\\Users\\fki1\\.spyder-py3\\autosave\\Flood_det_rgb_plot_application_of_sortindex-1.py'
```

## Versions

* Spyder version: 5.3.3  (conda)
* Python version: 3.8.16 64-bit
* Qt version: 5.15.2
* PyQt5 version: 5.15.7
* Operating System: Windows 10

### Dependencies

```
# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  2.2.1 (OK)
cookiecutter >=1.6.0          :  1.7.3 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.31.1;<8.0.0       :  7.31.1 (OK)
jedi >=0.17.2;<0.19.0         :  0.18.1 (OK)
jellyfish >=0.7               :  0.9.0 (OK)
jsonschema >=3.2.0            :  4.16.0 (OK)
keyring >=17.0.0              :  23.4.0 (OK)
nbconvert >=4.0               :  6.5.4 (OK)
numpydoc >=0.6.0              :  1.5.0 (OK)
paramiko >=2.4.0              :  2.8.1 (OK)
parso >=0.7.0;<0.9.0          :  0.8.3 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.9.0 (OK)
pygments >=2.0                :  2.11.2 (OK)
pylint >=2.5.0;<3.0           :  2.14.5 (OK)
pyls_spyder >=0.4.0           :  0.4.0 (OK)
pylsp >=1.5.0;<1.6.0          :  1.5.0 (OK)
pylsp_black >=1.2.0           :  1.2.1 (OK)
qdarkstyle >=3.0.2;<3.1.0     :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.2.2 (OK)
qtawesome >=1.0.2             :  1.2.2 (OK)
qtconsole >=5.3.2;<5.4.0      :  5.3.2 (OK)
qtpy >=2.1.0                  :  2.2.0 (OK)
rtree >=0.9.7                 :  0.9.7 (OK)
setuptools >=49.6.0           :  65.6.3 (OK)
sphinx >=0.6.6                :  5.0.2 (OK)
spyder_kernels >=2.3.3;<2.4.0 :  2.3.3 (OK)
textdistance >=4.2.0          :  4.2.1 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3             :  2.1.6 (OK)
zmq >=22.1.0                  :  23.2.0 (OK)

# Optional:
cython >=0.21                 :  None (NOK)
matplotlib >=3.0.0            :  3.6.2 (OK)
numpy >=1.7                   :  1.23.5 (OK)
pandas >=1.1.1                :  1.5.2 (OK)
scipy >=0.17.0                :  1.7.3 (OK)
sympy >=0.7.3                 :  None (NOK)
```

### Environment

<details><summary>Environment</summary>

```
# packages in environment at C:\ProgramData\Anaconda3\envs\radar:
#
# Name                    Version                   Build  Channel
_tflow_select             2.3.0                       mkl    anaconda
abseil-cpp                20211102.0           hd77b12b_0    anaconda
absl-py                   1.4.0            py38haa95532_0  
affine                    2.3.0              pyhd3eb1b0_0    anaconda
aiohttp                   3.8.3            py38h2bbff1b_0    anaconda
aiosignal                 1.2.0              pyhd3eb1b0_0    anaconda
alabaster                 0.7.12             pyhd3eb1b0_0    anaconda
anyio                     3.5.0            py38haa95532_0    anaconda
argon2-cffi               21.3.0             pyhd3eb1b0_0    anaconda
argon2-cffi-bindings      21.2.0           py38h2bbff1b_0    anaconda
arrow                     1.2.3            py38haa95532_0    anaconda
astroid                   2.11.7           py38haa95532_0    anaconda
astunparse                1.6.3                      py_0    anaconda
async-timeout             4.0.2            py38haa95532_0    anaconda
atomicwrites              1.4.0                      py_0    anaconda
attrs                     22.1.0           py38haa95532_0    anaconda
autopep8                  1.6.0              pyhd3eb1b0_1    anaconda
babel                     2.11.0           py38haa95532_0    anaconda
backcall                  0.2.0              pyhd3eb1b0_0    anaconda
bcrypt                    3.2.0            py38h2bbff1b_1    anaconda
beautifulsoup4            4.11.1           py38haa95532_0    anaconda
binaryornot               0.4.4              pyhd3eb1b0_1    anaconda
black                     22.6.0           py38haa95532_0    anaconda
blas                      1.0                         mkl    conda-forge
bleach                    4.1.0              pyhd3eb1b0_0    anaconda
blinker                   1.4              py38haa95532_0    anaconda
bottleneck                1.3.5            py38h080aedc_0    anaconda
brotli                    1.0.9                h2bbff1b_7    anaconda
brotli-bin                1.0.9                h2bbff1b_7    anaconda
brotlipy                  0.7.0           py38h294d835_1004    conda-forge
bzip2                     1.0.8                he774522_0    anaconda
c-ares                    1.19.0               h2bbff1b_0  
ca-certificates           2023.05.30           haa95532_0  
cachetools                4.2.2              pyhd3eb1b0_0    anaconda
certifi                   2023.5.7         py38haa95532_0  
cffi                      1.15.1           py38hd8c33c5_0    conda-forge
cfitsio                   3.470                h2bbff1b_7    anaconda
chardet                   4.0.0           py38haa95532_1003    anaconda
charset-normalizer        2.1.1              pyhd8ed1ab_0    conda-forge
click                     8.0.4            py38haa95532_0    anaconda
click-plugins             1.1.1              pyhd3eb1b0_0    anaconda
cligj                     0.7.2            py38haa95532_0    anaconda
cloudpickle               2.2.1              pyhd8ed1ab_0    conda-forge
colorama                  0.4.6            py38haa95532_0    anaconda
comm                      0.1.2            py38haa95532_0    anaconda
contourpy                 1.0.5            py38h59b6b97_0    anaconda
cookiecutter              1.7.3              pyhd3eb1b0_0    anaconda
cryptography              37.0.4           py38hb7941b4_0    conda-forge
curl                      7.87.0               h2bbff1b_0    anaconda
cycler                    0.11.0             pyhd3eb1b0_0    anaconda
cytoolz                   0.12.0           py38h294d835_0    conda-forge
dask-core                 2023.1.1           pyhd8ed1ab_0    conda-forge
debugpy                   1.5.1            py38hd77b12b_0    anaconda
decorator                 5.1.1              pyhd3eb1b0_0    anaconda
defusedxml                0.7.1              pyhd3eb1b0_0    anaconda
diff-match-patch          20200713           pyhd3eb1b0_0    anaconda
dill                      0.3.6            py38haa95532_0    anaconda
docutils                  0.18.1           py38haa95532_3    anaconda
entrypoints               0.4              py38haa95532_0    anaconda
enum34                    1.1.10           py38h32f6830_2    conda-forge
expat                     2.4.9                h6c2663c_0    anaconda
fftw                      3.3.9                h2bbff1b_1    anaconda
fiona                     1.8.22           py38hac22706_0    anaconda
flake8                    4.0.1              pyhd3eb1b0_1    anaconda
flatbuffers               2.0.0                h6c2663c_0    anaconda
flit-core                 3.6.0              pyhd3eb1b0_0    anaconda
fonttools                 4.25.0             pyhd3eb1b0_0    anaconda
freetype                  2.12.1               ha860e81_0    anaconda
freexl                    1.0.6                h2bbff1b_0    anaconda
frozenlist                1.3.3            py38h2bbff1b_0    anaconda
fsspec                    2023.1.0           pyhd8ed1ab_0    conda-forge
gast                      0.4.0              pyhd3eb1b0_0    anaconda
gdal                      3.0.2                    pypi_0    pypi
geopandas                 0.9.0                      py_1    anaconda
geopandas-base            0.9.0                      py_1    anaconda
geos                      3.8.0                h33f27b4_0    anaconda
geotiff                   1.7.0                h4545760_0    anaconda
giflib                    5.2.1                h8cc25b3_3  
glib                      2.69.1               h5dc1a3c_2    anaconda
google-auth               2.6.0              pyhd3eb1b0_0    anaconda
google-auth-oauthlib      0.4.4              pyhd3eb1b0_0    anaconda
google-pasta              0.2.0              pyhd3eb1b0_0    anaconda
grpc-cpp                  1.48.2               hf108199_0  
grpcio                    1.48.2           py38hf108199_0  
gst-plugins-base          1.18.5               h9e645db_0    anaconda
gstreamer                 1.18.5               hd78058f_0    anaconda
h5py                      3.7.0            py38h3de5c98_0    anaconda
hdf4                      4.2.13               h712560f_2    anaconda
hdf5                      1.10.6               h1756f20_1    anaconda
icc_rt                    2022.1.0             h6049295_2    anaconda
icu                       58.2                 ha925a31_3    anaconda
idna                      3.4                pyhd8ed1ab_0    conda-forge
imagecodecs-lite          2019.12.3        py38hbdcd294_5    conda-forge
imageio                   2.25.0             pyh24c5eb1_0    conda-forge
imagesize                 1.4.1            py38haa95532_0    anaconda
importlib-metadata        4.11.3           py38haa95532_0    anaconda
importlib_metadata        4.11.3               hd3eb1b0_0    anaconda
importlib_resources       5.2.0              pyhd3eb1b0_1    anaconda
inflection                0.5.1            py38haa95532_0    anaconda
intel-openmp              2021.4.0          haa95532_3556    anaconda
intervaltree              3.1.0              pyhd3eb1b0_0    anaconda
ipykernel                 6.19.2           py38hd4e2768_0    anaconda
ipympl                    0.8.7              pyhd3eb1b0_0  
ipython                   7.31.1           py38haa95532_1    anaconda
ipython_genutils          0.2.0              pyhd3eb1b0_1    anaconda
ipywidgets                7.6.5              pyhd3eb1b0_1  
isort                     5.9.3              pyhd3eb1b0_0    anaconda
jedi                      0.18.1           py38haa95532_1    anaconda
jellyfish                 0.9.0            py38h2bbff1b_0    anaconda
jinja2                    3.1.2            py38haa95532_0    anaconda
jinja2-time               0.2.0              pyhd3eb1b0_3    anaconda
joblib                    1.1.1            py38haa95532_0    anaconda
jpeg                      9e                   h2bbff1b_0    anaconda
jsonschema                4.16.0           py38haa95532_0    anaconda
jupyter_client            7.4.8            py38haa95532_0    anaconda
jupyter_core              5.1.1            py38haa95532_0    anaconda
jupyter_server            1.23.4           py38haa95532_0    anaconda
jupyterlab_pygments       0.1.2                      py_0    anaconda
jupyterlab_widgets        3.0.5            py38haa95532_0  
kealib                    1.4.14               hde4a422_1    anaconda
keras                     2.10.0           py38haa95532_0    anaconda
keras-preprocessing       1.1.2              pyhd3eb1b0_0    anaconda
keyring                   23.4.0           py38haa95532_0    anaconda
kiwisolver                1.4.4            py38hd77b12b_0    anaconda
krb5                      1.19.4               h5b6d351_0    anaconda
lazy-object-proxy         1.6.0            py38h2bbff1b_0    anaconda
lerc                      3.0                  hd77b12b_0    anaconda
libbrotlicommon           1.0.9                h2bbff1b_7    anaconda
libbrotlidec              1.0.9                h2bbff1b_7    anaconda
libbrotlienc              1.0.9                h2bbff1b_7    anaconda
libclang                  12.0.0          default_h627e005_2    anaconda
libcurl                   7.87.0               h86230a5_0    anaconda
libdeflate                1.8                  h2bbff1b_5    anaconda
libffi                    3.4.2                hd77b12b_6    anaconda
libgdal                   3.0.2                hc12e7b7_6    anaconda
libiconv                  1.16                 h2bbff1b_2    anaconda
libnetcdf                 4.8.1                h6685c40_2    anaconda
libogg                    1.3.5                h2bbff1b_1    anaconda
libpng                    1.6.37               h2a8f88b_0    anaconda
libpq                     12.9                 hb652d5d_3    anaconda
libprotobuf               3.20.3               h23ce68f_0    anaconda
libsodium                 1.0.18               h62dcd97_0    anaconda
libspatialindex           1.9.3                h6c2663c_0    anaconda
libspatialite             4.3.0a              h14feca5_21    anaconda
libssh2                   1.10.0               hcd4344a_0    anaconda
libtiff                   4.5.0                h8a3f274_0    anaconda
libvorbis                 1.3.7                he774522_0    anaconda
libwebp                   1.2.4                h2bbff1b_0    anaconda
libwebp-base              1.2.4                h2bbff1b_0    anaconda
libxml2                   2.9.14               h0ad7f3c_0    anaconda
libxslt                   1.1.35               h2bbff1b_0    anaconda
libzip                    1.8.0                h49b8836_0    anaconda
locket                    1.0.0              pyhd8ed1ab_0    conda-forge
lxml                      4.9.1            py38h1985fb9_0    anaconda
lz4-c                     1.9.4                h2bbff1b_0    anaconda
m2w64-expat               2.1.1                         2    conda-forge
m2w64-gcc-libgfortran     5.3.0                         6    conda-forge
m2w64-gcc-libs            5.3.0                         7    conda-forge
m2w64-gcc-libs-core       5.3.0                         7    conda-forge
m2w64-gettext             0.19.7                        2    conda-forge
m2w64-gmp                 6.1.0                         2    conda-forge
m2w64-libiconv            1.14                          6    conda-forge
m2w64-libwinpthread-git   5.0.0.4634.697f757               2    conda-forge
m2w64-xz                  5.2.2                         2    conda-forge
mapclassify               2.5.0            py38haa95532_0    anaconda
markdown                  3.4.1            py38haa95532_0    anaconda
markupsafe                2.1.1            py38h2bbff1b_0    anaconda
matplotlib                3.6.2            py38haa95532_0    anaconda
matplotlib-base           3.6.2            py38h1094b8e_0    anaconda
matplotlib-inline         0.1.6            py38haa95532_0    anaconda
mccabe                    0.7.0              pyhd3eb1b0_0    anaconda
mistune                   0.8.4           py38he774522_1000    anaconda
mkl                       2021.4.0           haa95532_640    anaconda
mkl-service               2.4.0            py38h2bbff1b_0    anaconda
mkl_fft                   1.3.1            py38h277e83a_0    anaconda
mkl_random                1.2.2            py38hf11a4ad_0    anaconda
msys2-conda-epoch         20160418                      1    conda-forge
multidict                 6.0.2            py38h2bbff1b_0    anaconda
munch                     2.5.0              pyhd3eb1b0_0    anaconda
munkres                   1.1.4                      py_0    anaconda
mypy_extensions           0.4.3            py38haa95532_1    anaconda
nbclassic                 0.4.8            py38haa95532_0    anaconda
nbclient                  0.5.13           py38haa95532_0    anaconda
nbconvert                 6.5.4            py38haa95532_0    anaconda
nbformat                  5.7.0            py38haa95532_0    anaconda
nest-asyncio              1.5.6            py38haa95532_0    anaconda
networkx                  2.8.4            py38haa95532_0    anaconda
notebook                  6.5.2            py38haa95532_0    anaconda
notebook-shim             0.2.2            py38haa95532_0    anaconda
numexpr                   2.8.4            py38h5b0cc5e_0    anaconda
numpy                     1.23.5           py38h3b20f71_0    anaconda
numpy-base                1.23.5           py38h4da318b_0    anaconda
numpydoc                  1.5.0            py38haa95532_0    anaconda
oauthlib                  3.2.2            py38haa95532_0  
opencv-python             4.8.0.74                 pypi_0    pypi
openjpeg                  2.4.0                h4fc8c34_0    anaconda
openssl                   1.1.1u               h2bbff1b_0  
opt_einsum                3.3.0              pyhd3eb1b0_1    anaconda
packaging                 22.0             py38haa95532_0    anaconda
pandas                    1.5.2            py38hf11a4ad_0    anaconda
pandocfilters             1.5.0              pyhd3eb1b0_0    anaconda
paramiko                  2.8.1              pyhd3eb1b0_0    anaconda
parso                     0.8.3              pyhd3eb1b0_0    anaconda
partd                     1.3.0              pyhd8ed1ab_0    conda-forge
pathlib                   1.0.1            py38haa244fe_7    conda-forge
pathspec                  0.10.3           py38haa95532_0    anaconda
pcre                      8.45                 hd77b12b_0    anaconda
pexpect                   4.8.0              pyhd3eb1b0_3    anaconda
pickleshare               0.7.5           pyhd3eb1b0_1003    anaconda
pillow                    9.3.0            py38hdc2b20a_1    anaconda
pip                       22.3.1           py38haa95532_0    anaconda
pkgutil-resolve-name      1.3.10           py38haa95532_0    anaconda
platformdirs              2.5.2            py38haa95532_0    anaconda
pluggy                    1.0.0            py38haa95532_1    anaconda
ply                       3.11                     py38_0    anaconda
poyo                      0.5.0              pyhd3eb1b0_0    anaconda
proj                      6.2.1                h3758d61_0    anaconda
prometheus_client         0.14.1           py38haa95532_0    anaconda
prompt-toolkit            3.0.36           py38haa95532_0    anaconda
protobuf                  3.20.3           py38hd77b12b_0    anaconda
psutil                    5.9.0            py38h2bbff1b_0    anaconda
ptyprocess                0.7.0              pyhd3eb1b0_2    anaconda
pyasn1                    0.4.8              pyhd3eb1b0_0    anaconda
pyasn1-modules            0.2.8                      py_0    anaconda
pycodestyle               2.8.0              pyhd3eb1b0_0    anaconda
pycparser                 2.21               pyhd8ed1ab_0    conda-forge
pydocstyle                6.1.1              pyhd3eb1b0_0    anaconda
pyflakes                  2.4.0              pyhd3eb1b0_0    anaconda
pygments                  2.11.2             pyhd3eb1b0_0    anaconda
pyjwt                     2.4.0            py38haa95532_0    anaconda
pylint                    2.14.5           py38haa95532_0    anaconda
pyls-spyder               0.4.0              pyhd3eb1b0_0    anaconda
pynacl                    1.5.0            py38h8cc25b3_0    anaconda
pyopenssl                 22.0.0             pyhd8ed1ab_1    conda-forge
pyparsing                 3.0.9            py38haa95532_0    anaconda
pyproj                    2.6.1.post1      py38h593ac45_1    anaconda
pyqt                      5.15.7           py38hd77b12b_0    anaconda
pyqt5-sip                 12.11.0          py38hd77b12b_0    anaconda
pyqtwebengine             5.15.7           py38hd77b12b_0    anaconda
pyrsistent                0.18.0           py38h196d8e1_0    anaconda
pyshp                     2.3.1              pyhd8ed1ab_0    conda-forge
pysocks                   1.7.1              pyh0701188_6    conda-forge
python                    3.8.16               h6244533_2    anaconda
python-dateutil           2.8.2              pyhd3eb1b0_0    anaconda
python-fastjsonschema     2.16.2           py38haa95532_0    anaconda
python-flatbuffers        2.0                pyhd3eb1b0_0    anaconda
python-lsp-black          1.2.1            py38haa95532_0    anaconda
python-lsp-jsonrpc        1.0.0              pyhd3eb1b0_0    anaconda
python-lsp-server         1.5.0            py38haa95532_0    anaconda
python-slugify            5.0.2              pyhd3eb1b0_0    anaconda
python_abi                3.8                      2_cp38    conda-forge
pytz                      2022.7           py38haa95532_0    anaconda
pywavelets                1.4.1            py38h2bbff1b_0    anaconda
pywin32                   305              py38h2bbff1b_0    anaconda
pywin32-ctypes            0.2.0                 py38_1000    conda-forge
pywinpty                  2.0.2            py38h5da7b33_0    anaconda
pyyaml                    6.0              py38h294d835_4    conda-forge
pyzmq                     23.2.0           py38hd77b12b_0    anaconda
qdarkstyle                3.0.2              pyhd3eb1b0_0    anaconda
qstylizer                 0.2.2            py38haa95532_0    anaconda
qt-main                   5.15.2               he8e5bd7_7    anaconda
qt-webengine              5.15.9               hb9a9bb5_5    anaconda
qtawesome                 1.2.2            py38haa95532_0    anaconda
qtconsole                 5.3.2            py38haa95532_0    anaconda
qtpy                      2.2.0            py38haa95532_0    anaconda
qtwebkit                  5.212                h3ad3cdb_4    anaconda
rasterio                  1.2.10           py38h17c1fa0_0    anaconda
rasterstats               0.17.0             pyhd8ed1ab_0    conda-forge
re2                       2022.04.01           hd77b12b_0    anaconda
requests                  2.28.2             pyhd8ed1ab_0    conda-forge
requests-oauthlib         1.3.0                      py_0    anaconda
rope                      0.22.0             pyhd3eb1b0_0    anaconda
rsa                       4.7.2              pyhd3eb1b0_1    anaconda
rtree                     0.9.7            py38h2eaa2aa_1    anaconda
scikit-image              0.19.3           py38hd77b12b_1    anaconda
scikit-learn              1.2.0            py38hd77b12b_1    anaconda
scipy                     1.7.3            py38h0a974cb_0    anaconda
seaborn                   0.12.2           py38haa95532_0    anaconda
send2trash                1.8.0              pyhd3eb1b0_1    anaconda
setuptools                65.6.3           py38haa95532_0    anaconda
shapely                   1.8.4            py38h9064783_0    anaconda
simplejson                3.17.6           py38h294d835_1    conda-forge
sip                       6.6.2            py38hd77b12b_0    anaconda
six                       1.16.0             pyhd3eb1b0_1    anaconda
snappy                    1.1.9                h6c2663c_0    anaconda
sniffio                   1.2.0            py38haa95532_1    anaconda
snowballstemmer           2.2.0              pyhd3eb1b0_0    anaconda
snuggs                    1.4.7              pyhd3eb1b0_0    anaconda
sortedcontainers          2.4.0              pyhd3eb1b0_0    anaconda
soupsieve                 2.3.2.post1      py38haa95532_0    anaconda
spectral                  0.23.1             pyh1a96a4e_0    conda-forge
sphinx                    5.0.2            py38haa95532_0    anaconda
sphinxcontrib-applehelp   1.0.2              pyhd3eb1b0_0    anaconda
sphinxcontrib-devhelp     1.0.2              pyhd3eb1b0_0    anaconda
sphinxcontrib-htmlhelp    2.0.0              pyhd3eb1b0_0    anaconda
sphinxcontrib-jsmath      1.0.1              pyhd3eb1b0_0    anaconda
sphinxcontrib-qthelp      1.0.3              pyhd3eb1b0_0    anaconda
sphinxcontrib-serializinghtml 1.1.5              pyhd3eb1b0_0    anaconda
spyder                    5.3.3            py38haa95532_0    anaconda
spyder-kernels            2.3.3            py38haa95532_0    anaconda
sqlite                    3.40.1               h2bbff1b_0    anaconda
tensorboard               2.10.0           py38haa95532_0    anaconda
tensorboard-data-server   0.6.1            py38haa95532_0    anaconda
tensorboard-plugin-wit    1.8.1            py38haa95532_0    anaconda
tensorflow                2.10.0          mkl_py38ha5c4042_0    anaconda
tensorflow-base           2.10.0          mkl_py38h6a7f48e_0    anaconda
tensorflow-estimator      2.10.0           py38haa95532_0    anaconda
termcolor                 2.1.0            py38haa95532_0    anaconda
terminado                 0.17.1           py38haa95532_0    anaconda
text-unidecode            1.3                pyhd3eb1b0_0    anaconda
textdistance              4.2.1              pyhd3eb1b0_0    anaconda
threadpoolctl             2.2.0              pyh0d69192_0    anaconda
three-merge               0.1.1              pyhd3eb1b0_0    anaconda
tifffile                  2019.7.26.2              py38_0    conda-forge
tiledb                    2.3.3                h3649cd2_2    anaconda
tinycss2                  1.2.1            py38haa95532_0    anaconda
tk                        8.6.12               h2bbff1b_0    anaconda
toml                      0.10.2             pyhd3eb1b0_0    anaconda
tomli                     2.0.1            py38haa95532_0    anaconda
tomlkit                   0.11.1           py38haa95532_0    anaconda
toolz                     0.12.0             pyhd8ed1ab_0    conda-forge
tornado                   6.2              py38h2bbff1b_0    anaconda
tqdm                      4.65.0           py38hd4e2768_0  
traitlets                 5.7.1            py38haa95532_0    anaconda
typing-extensions         4.4.0            py38haa95532_0    anaconda
typing_extensions         4.4.0            py38haa95532_0    anaconda
ujson                     5.4.0            py38hd77b12b_0    anaconda
unidecode                 1.2.0              pyhd3eb1b0_0    anaconda
urllib3                   1.26.14            pyhd8ed1ab_0    conda-forge
vc                        14.2                 h21ff451_1    anaconda
vs2015_runtime            14.27.29016          h5e58377_2    anaconda
watchdog                  2.1.6            py38haa95532_0    anaconda
wcwidth                   0.2.5              pyhd3eb1b0_0    anaconda
webencodings              0.5.1                    py38_1    anaconda
websocket-client          0.58.0           py38haa95532_4    anaconda
werkzeug                  2.2.3            py38haa95532_0  
whatthepatch              1.0.2            py38haa95532_0    anaconda
wheel                     0.37.1             pyhd3eb1b0_0    anaconda
widgetsnbextension        3.5.2            py38haa95532_0  
win_inet_pton             1.1.0              pyhd8ed1ab_6    conda-forge
wincertstore              0.2              py38haa95532_2    anaconda
winpty                    0.4.3                         4    conda-forge
wrapt                     1.14.1           py38h2bbff1b_0    anaconda
xarray                    2022.11.0        py38haa95532_0    anaconda
xerces-c                  3.2.4                hd77b12b_0    anaconda
xz                        5.2.10               h8cc25b3_1    anaconda
yaml                      0.2.5                h8ffe710_2    conda-forge
yapf                      0.31.0             pyhd3eb1b0_0    anaconda
yarl                      1.8.1            py38h2bbff1b_0    anaconda
zeromq                    4.3.4                hd77b12b_0    anaconda
zipp                      3.11.0           py38haa95532_0    anaconda
zlib                      1.2.13               h8cc25b3_0    anaconda
zstd                      1.5.2                h19a0ad4_0    anaconda

```
</details>

**Comment by ccordoba12** [comment]
Hey @kasitif, thanks for reporting. I think this problem is fixed in our latest version available in Anaconda (5.4.3). Please update by closing Spyder, opening the Anaconda Prompt and running there the commands mentioned in [our documentation](http://docs.spyder-ide.org/current/installation.html#updating-with-anaconda).

Let us know if that works for you.

**Comment by ccordoba12** [comment]
Closing due to lack of response.
